spec/prodecuralSpec.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace procedural\spec;
+
+use function prodecural\echoScript;
+
+describe('procedural', function () {
+  describe('->echoScript()', function () {
+    it('echos input string inside a script tag', function () {
+      $closure = function() { echoScript('baz'); };
+      expect($closure)->toEcho('<script>baz</script>');
+    });
```

Comment:
Once I'm unblocked I'll also go back and smooth out the style here as I see you're using more whitespace and indentation in other files.